R/adult.R

```diff
@@ -0,0 +1,45 @@
+#' @title Adult Dataset
+#'
+#' @name adult
+#' @aliases adult_test
+#' @aliases adult_train
+#'
+#' @description
+#' Dataset used to predict whether income exceeds $50K/yr based on census data. Also known as "Census Income" dataset
+#' Train dataset contains 13 features and 30178 observations.
+#' Test dataset contains 13 features and 15315 observations.
+#' Target column is "Target".
```

Comment:
Can you write half a sentence more about what the two levels mean? It is already in L:8, but maybe not immediately clear.